sound/soc/sof/hw-spi.c

```diff
@@ -167,20 +167,6 @@ static irqreturn_t spi_irq_handler(int irq __maybe_unused, void *context)
 	return IRQ_NONE;
 }
 
-static irqreturn_t spi_irq_thread(int irq __maybe_unused, void *context __maybe_unused)
```

Comment:
I am probably getting on your nerves but even this needs to be split in 3. My topic/sof-def-rebase have haswell, SKL and SPI added in 3 separate steps.